cis_v110/section_2.sp

```diff
@@ -14,7 +14,11 @@ benchmark "cis_v110_2" {
     control.cis_v110_2_4,
     control.cis_v110_2_5,
   ]
-  tags          = local.cis_v110_1_common_tags
+
+  tags = merge(local.cis_v110_2_common_tags, {
+    service = "OCI/Networking"
```

Comment:
Should this be `OCI/VCN` instead? We should be consistent with what we have in the OCI Insights mod (or if that mod's tags are incorrect, we should update those).